osFiles/macOS/20x - 11.x/20C5061b.json

```diff
@@ -94,8 +94,6 @@
                 "MacBookAir8,2",
                 "MacBookAir9,1",
                 "MacBookAir10,1",
-                "MacBookPro11,1-Late-2013",
```

Comment:
Intentional?

Reply:
Triple-checked, and this is around where Big Sur was bricking the 13" MBP